recipe/meta.yaml

```diff
@@ -69,7 +69,7 @@ requirements:
         - qt
         - xerces-c
         - zlib
-        - boost-cpp
+        - libboost
```

Comment:
This isn't necessary; `libboost-devel` will ensure that the right `libboost` is available at runtime through a so-called run-export.